src/js/sync-to-drive.js

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env node
+/* eslint-disable node/shebang */
+
+const {execSync, spawn} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const {pid: PID, platform: PLATFORM} = process;
+
+//This script needs to be run from within the photobooth directory
+const API_DIR_NAME = 'api';
+const API_FILE_NAME = 'config.php';
+
+const getConfigFromPHP = () => {
+    const cmd = 'cd ' + API_DIR_NAME + ' && php ./' + API_FILE_NAME;
+
+    try {
+        const stdout = execSync(cmd).toString();
+
+        return JSON.parse(stdout.slice(stdout.indexOf('{'), -1));
+    } catch (err) {
+        console.log('Sync-To-Drive server [', PID, ']: ERROR: Couldnt get config from PHP', err);
+    }
+
+    return null;
+};
+
+const parseConfig = (config) => {
+    if (!config) {
+        return null;
+    }
+
+    try {
+        return {
+            dataAbsPath: config.foldersAbs.data,
+            drives: [config.synctodrive_targets.split(';')]
```

Comment:
Now its putting an array into an array :D
Think of `"h e l l o".split(' ')` => `[ [ 'h', 'a', 'l', 'l', 'o' ] ]`